SPLASH/src/game/states/GameState.cpp

```diff
@@ -844,6 +866,56 @@ bool GameState::renderImgui(float dt) {
 	//ImGui::End();
 	//ECS::Instance()->getSystem<AnimationSystem>()->updateHands(lPos, rPos, lRot, rRot);
 
+
+
+	if (m_isInKillCamMode) {
+		const unsigned int height = m_app->getWindow()->getWindowHeight() / 6;
+
+		ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0.f);
+		ImGui::PushStyleColor(ImGuiCol_WindowBg, ImVec4(0.f, 0.f, 0.f, 0.7f));
+		if (ImGui::Begin("##KILLCAMBACKGROUNDTOP", nullptr, m_backgroundOnlyflags)) {
+			ImGui::SetWindowSize(ImVec2(m_app->getWindow()->getWindowWidth(), height));
+			ImGui::SetWindowPos(ImVec2(0, 0));
+		}
+		ImGui::End();
+		ImGui::PopStyleColor(1);
+		ImGui::PopStyleVar(1);
+
+
+		ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0.f);
```

Comment:
Is it necessary to pop then push the same styleVar and styleColor for "##KILLCAMBACKGROUNDTOP" and "##KILLCAMBACKGROUNDBOTTOM"?